regression/tests/common/test_html_component.py

```diff
@@ -0,0 +1,115 @@
+"""
+End to end tests for HTML Componenets
+"""
+import os
+from bok_choy.web_app_test import WebAppTest
+from regression.pages.studio.course_outline_page import (
+    CourseOutlinePageExtended
+)
+from regression.pages.studio.unit_page import UnitPageExtended
+from regression.pages.studio.login_studio import StudioLogin
+from regression.pages.studio.studio_home import DashboardPageExtended
+from regression.pages.lms.login_lms import LmsLogin
+from edxapp_acceptance.pages.lms.courseware import CoursewarePage
+from regression.tests.helpers import (
+    LoginHelper, get_course_info, get_data_id_of_component
+)
+
+from edxapp_acceptance.pages.studio.utils import add_components
+
+
+class StudioLmsHTMLTest(WebAppTest):
+    """
+    HTML Components tests that require lms verification with studio
+    """
+
+    DEMO_COURSE_USER = os.environ.get('USER_LOGIN_EMAIL')
+    DEMO_COURSE_PASSWORD = os.environ.get('USER_LOGIN_PASSWORD')
+
+    def setUp(self):
+        """
+        Initialize the page object
+        """
+        super(StudioLmsHTMLTest, self).setUp()
+        # Login to Lms first to avoid authentication problems
+        self.login_page = LmsLogin(self.browser)
+        LoginHelper.login(self.login_page)
```

Comment:
It does I think. 
What this is doing is, on stage, when we have to use View Live (navigating from Studio to LMS), windows authentication box appears, which breaks the test. To avoid that we login to LMS first, then on Studio, perform our tasks and then if we click View Live (navigate from Studio to LMS, it directly goes to the page without asking for windows authentication. By windows authentication I mean stage server security.